hack/update-bazel.sh

```diff
@@ -22,5 +22,8 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 git config http.https://gopkg.in.followRedirects true
 
+# Remove generated files prior to running gazel.
+rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
```

Comment:
Will this exist forever?  This means that any time we run gazel, we will have to rebuild all generated files, even if net nothing changed...

Or does this come with a TODO?